doc/users/dflt_style_changes.rst

```diff
@@ -1065,6 +1065,17 @@ Z-order
 
 
 
+``ScalarFormatter`` tick label formatting with offsets
+======================================================
+
+With the default of ``rcParams['axes.formatter.useoffset'] = True``,
+an offset will be used when it will save 4 or more digits.  This can
```

Comment:
It's save in the third sense (of [this dictionary](http://www.merriam-webster.com/dictionary/save)):

> 3 a :  to make unnecessary :  avoid <it saves an hour's driving>